spec/frameSpec.js

```diff
@@ -0,0 +1,48 @@
+describe('Frame', function() {
+    beforeEach(function() {
+        frame = new Frame;
+    });
+
+    var completedFrame = function() {
```

Comment:
Love the addition of this function, makes the tests really nice and readable.
Maybe it could be separated into a webHelper file? (i've not tried that myself, i wonder if you would just require the webhelper through the spec runner)

Reply:
I did look into that, but from what I could see it isn't super easy to implement a helper in jasmine. You need to add a specific node but I read that it isn't as useful as rspec helpers. Or so the 10 mins of research told me haha